src/sentry/conf/server.py

```diff
@@ -814,6 +814,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "sentry.uptime.detectors.tasks",
     "sentry.uptime.subscriptions.tasks",
     "sentry.integrations.vsts.tasks",
+    "sentry.integrations.vsts.tasks.kickoff_subscription_check",
```

Comment:
It seems like this is needed to pass the test `tests/sentry/celery/test_app.py`, even though the 'folder' with an __init__ is specified above (?). Is my understanding of celery off that, celery would be able to find this task (even though the test said otherwise) ?

Reply:
Yeah, the test is naive and assumes that a module import doesn't pull in all of the child modules (which is a safe assumption to make imo). Additional celery imports don't hurt anything.